src/service/resources-impl.js

```diff
@@ -778,6 +778,29 @@ export class Resources {
   }
 
   /**
+   * Return a promise that requests runtime to collapse this element.
+   * The runtime will schedule this request and first attempt to resize
+   * the element to height and width 0. If success runtime will set element
+   * display to none, and notify element owner of this collapse.
+   * @param {!Element} element
+   * @return {!Promise}
+   */
+  attemptCollapse(element) {
+    return new Promise((resolve, reject) => {
```

Comment:
That's the thing that I want to fix in this PR.

`#attemptChangeSize` success, resources manager does some calculation to adjust scrollTop, we then immediately collapse the element the previous measurement gives incorrect value which result in a page jump.